src/entt/entity/storage.hpp

```diff
@@ -833,6 +977,14 @@ class basic_storage<Type, Entity, Allocator, std::enable_if_t<component_traits<T
         return allocator_type{base_type::get_allocator()};
     }
 
+    /**
+     * @brief Returns the associated traits.
+     * @return The associated traits.
+     */
+    [[nodiscard]] constexpr traits_type const &get_traits() const noexcept {
```

Comment:
Does it make sense to have traits on an empty type? It sounds like an unwanted side effect. 🤔

Reply:
This method exists for consistency with the other specializations.